src/emuvim/examples/sonata_simple_topology.py

```diff
@@ -40,13 +40,14 @@
 from emuvim.api.rest.rest_api_endpoint import RestApiEndpoint
 from emuvim.api.sonata import SonataDummyGatekeeperEndpoint
 from mininet.node import RemoteController
+import os
 
 logging.basicConfig(level=logging.INFO)
 
 
 def create_topology1():
     # create topology
-    net = DCNetwork(controller=RemoteController, monitor=True, enable_learning=True)
+    net = DCNetwork(controller=RemoteController, monitor=False, enable_learning=True)
```

Comment:
Same here. We should not change default values of existing examples. Its better to simply create a new example. Otherwise old documentations and demos will break.